lib/src/anchor.dart

```diff
@@ -86,9 +86,11 @@ class Anchor {
       return _valueNames.entries.singleWhere((e) => e.value == name).key;
     } else {
       final regexp = RegExp(r'^\Anchor\(([^,]+), ([^\)]+)\)');
-      final matches = regexp.allMatches(name).first.group;
-      assert(matches != null, 'Bad Anchor format');
-      return Anchor(double.parse(matches(1)), double.parse(matches(2)));
+      final matches = regexp.allMatches(name).first.groups([1, 2]);
+      final x = matches[0];
+      final y = matches[1];
+      assert(x != null && y != null, 'bad anchor format');
```

Comment:
I don't find this very pretty but it's the way I found to work on the notnull world